<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtArteryTechOtherSetupsUI</class>
 <widget class="QDialog" name="QtArteryTechOtherSetupsUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtArteryTechOtherSetupsUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="InitialNumberOfPartitionLabel">
         <property name="text">
          <string>Initial Number of Partition:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="InitialNumberOfPartitionLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="TypeOfPartitionerLabel">
         <property name="text">
          <string>Type of Partitioner:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TypeOfPartitionerLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="DiagnalScalerLabel">
         <property name="text">
          <string>Diagnal Scaler:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="DiagnalScalerLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="PointBlockSolverLabel">
         <property name="text">
          <string>Point Block Solver:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PointBlockSolverLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="LogSummaryLabel">
         <property name="text">
          <string>Log Summary:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LogSummaryLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="OKPushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
